INSTALL

```diff
@@ -63,17 +63,20 @@ Check System Prerequisites
     running from a distro many of these are taken care of, if not here they
     are:
 
-     * platform must have an Intel® Communications Chipset 4xxx Series device
-        Use "lspci -d 8086:4940" to check PF devices exist.
-        Use "lspci -d 8086:4941" to check VF devices exist.
-     * OR platform must have an Intel® 401xx series device
-        Use "lspci -d 8086:4942" to check PF devices exist.
-        Use "lspci -d 8086:4943" to check VF devices exist.
-        VFs are only visible if all prerequisites below are satisfied.
+     * platform must have one of the following Intel® Communications devices:
+        4xxx  : Use "lspci -d 8086:4940" to check Physical Function (PF)
+                devices are present.
+        401xx : Use "lspci -d 8086:4942" to check PF devices are present.
+     	Note: Later, after "systemctl start qat" or "make install" steps, the
```

Comment:
Remove tabs from this and the next 2 lines.

Reply:
done